R/zcurve.R

```diff
@@ -0,0 +1,31 @@
+#' Create a curve comparing observed Z-scores to the WHO standard.
+#'
+#' @param zscore a numeric vector containing computed zscores
+#' @return a ggplot2 object
+#' @export
+#' @importFrom ggplot2 ggplot aes stat_function geom_density scale_x_continuous labs
+#' @examples
+#' dat <- rnorm(204) + runif(1) # slightly skewed
+#' zcurve(dat)
```

Comment:
Can you add an example that sets a title for plot? E.g. "Weight-for-Height z-scores"?

Reply:
Yup, see https://github.com/R4EPI/epireports/pull/25#issuecomment-454296280